### PR TITLE
fix(figma-svg): evaluate a graphics-layer block against the node's real box (#2615)

### PR DESCRIPTION
**Pull request by yschimke**
Two commits: the CI gap that let this class of bug survive four releases, and the #2615 fix it would have caught.

## 1. `ci:` run every module's unit tests

The named jobs cover a handful of modules by task path (`:renderer-android`, `:gradle-plugin`, `:cli` via its `build`, `:daemon:harness`, …). That left **66 modules with `src/test/kotlin` that no workflow ever invoked** — including `data/layoutinspector/{core,connector}`, which holds the `compose/figma-svg` export model and every one of its regression tests, and both daemon backends, which hold the end-to-end preview fixtures.

That's a large part of why #2853 and #2615 kept reopening against releases meant to have fixed them: fixtures were written, merged, and never ran anywhere.

Root-level task-name abbreviation fans `test` out to every project that has it. The render-driven `samples/*` tests are excluded — already covered by the dedicated render jobs, so running them here doubles a slow job rather than adding coverage. `--continue` reports every module in one pass.

Verified by dry run: the fan-out now includes `:data-layoutinspector-core:test`, `:data-layoutinspector-connector:test`, `:daemon:android:testDebugUnitTest` and `:daemon:desktop:test`, and every `-x` path resolves.

> This job has never run before, so it may surface pre-existing failures in modules that haven't been exercised. That's the point — but it means the first run is informational as much as it is a gate.

## 2. `fix:` the #2615 near-zero alpha

Your 0.19.11 comment isolated it exactly: every failing screen uses `TransformingLazyColumn` + `transformedHeight` + `SurfaceTransformation` and carries spurious `opacity="0.0"` / `"0.04"` groups, while the healthy standalone rows carry none. Confirmed in the published `episode-screen` SVG — one `0.0` and three `0.04`.

`evaluateLayerBlockAlpha` runs a lambda-form `graphicsLayer { … }` against a proxied `GraphicsLayerScope` and records what it assigns. Two defects compounded:

1. **Getters answered with the type's zero**, so `size` was `Size.Zero`. Wear's transform computes item alpha from how the slot was scaled — it divided into a 0×0 box.
2. **`size` and `center` are value classes** (`Size`/`Offset`, packed over a `Long`), so Kotlin mangles their accessors to `getSize-NH-jbRc`. Deriving the property with `removePrefix("get")` yielded `size-NH-jbRc`, matching neither the recorded assignments nor the identity defaults — so even a correct default could never have been returned, and mangled *setters* were silently dropped from the recording too.

The node's placed size is now threaded in from the modifier's coordinates and answered as `size` (with `center` derived from it), and accessor names have their value-class mangling stripped before lookup.

Defect 2 is the interesting one: it means the existing `GRAPHICS_LAYER_DEFAULTS` entries were only ever reachable for non-value-class properties. `alpha`/`scaleX`/`scaleY` are plain floats so they worked; anything packed did not.

## Test plan

- New `GraphicsLayerBlockGeometryTest` (6 tests) in `:data-layoutinspector-connector`. The two asserting a size-derived alpha **failed before the mangling fix and pass after** — that's what pinned defect 2.
- Preserved behaviours pinned alongside: a detached node keeps the identity defaults; a block ignoring geometry (the closed-over animation value of #2853) reads back unchanged; `alpha *= …` still starts from 1f.
- `:data-layoutinspector-connector:test` (142) and `:data-layoutinspector-core:test` green.
- **Not verified end-to-end.** Jetcaster Wear isn't reproducible in this container — no Wear render path here, and neither daemon backend runs (Robolectric SIGABRTs, skiko can't load `libGL.so.1`). The fix is verified at the evaluator level; confirming the scores move needs a real Wear render.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_015GpzckZtKxBbgy3SSE6LSn

---
_Generated by [Claude Code](https://claude.ai/code/session_015GpzckZtKxBbgy3SSE6LSn)_

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -78,6 +78,59 @@ jobs:
...
+  # reopening against releases that were supposed to have fixed them.
+  #
+  # Root-level task-name abbreviation fans `test` out to every project that has it. The `samples/*`
+  # modules are excluded: their tests drive real renders and are already covered by the dedicated
+  # render jobs (`Build Samples`, `Render + pixel-test Wear sample previews`), so running them here
+  # would double a slow job rather than add coverage.
+  #
+  # `--continue` so one failing module reports every other module's result in the same run instead
+  # of masking them — the point of the job is to know the true state, not to stop at the first red.
+  module-unit-tests:
```
[the remark this answers is not in the document]
Comment (yschimke) [reply .github/workflows/ci.yml]: Right on both counts, and worse than it looked from the diff — I confirmed the parsed structure before fixing:

```
renderer-android-tests -> NULL
module-unit-tests      -> name: "Renderer Android Unit Tests"
```

So neither job ran as intended: `Module Unit Tests` never registered as a check at all (which is why it was missing from this PR's check list), and `Renderer Android Unit Tests` was actually the new job wearing the renderer's name via duplicate-key last-wins.

Fixed in b356378 by moving the block after the complete `renderer-android-tests` mapping.

Worth recording why my validation missed it: I checked with `yaml.safe_load`, which resolves duplicate keys silently, so the file parsed cleanly into the wrong shape and looked fine. I've re-validated with a loader that *rejects* duplicate mapping keys — that's what catches this class of splice — and both jobs now resolve to distinct mappings with their own names and steps.

---
_Generated by [Claude Code](https://claude.ai/code)_
